README.md

```diff
@@ -2,9 +2,39 @@
 
 > This package includes the Babel preset used by Enact.
 
+## Usage
+
+Reference this module from your `babel.config.js`.
+
+```js
+module.exports = require('babel-preset-enact');
+```
+
+For more information (including `babel.config.js` setup), please see the [babel docs](https://babeljs.io/docs/config-files).
```

Comment:
Try not to use ( ) in this phase. Would you rewrite this, please?

Reply:
OK